src/languages/es.ts

```diff
@@ -2721,6 +2721,18 @@ export default {
             errorDescriptionPartTwo: 'contacta con el conserje',
             errorDescriptionPartThree: 'por ayuda.',
         },
+        restrictedAction: {
+            restricted: 'Restringido',
+            actionsAreCurrentlyRestricted: ({ workspaceName }) => `Los gastos para ${workspaceName} están actualmente restringidos`,
```

Comment:
@pac-guerreiro Please resolve this comment once you update the spanish translation with the correct text.

Reply:
Los gastos para el espacio de trabajo {workspace_name} están actualmente restringidos

😁

Reply:
@Gonals Muchas gracias! 🙏

Reply:
Ah coming back here - the copy for this has an additional change that was made in the design doc reviews -- 

Instead of "Los gastos para el espacio de trabajo {workspace_name} están actualmente restringidos", we want to change the copy to "Las acciones en el espacio de trabajo {workspace_name} están actualmente restringidas" 

@Gonals - are you able to confirm if that is the correct formatting to say, "Actions on the {workspace_name} workspace are currently restricted" ? I think we'd also need to change the English file for this as well? As initially the design doc had "Expenses to {workspace_name} are currently restricted."

Reply:
English file is correct in the PR it seems: 

<img width="945" alt="image" src="https://github.com/Expensify/App/assets/16232057/eb4e9459-5e3a-45cb-b970-69a70c85ca11">

Reply:
> Ah coming back here - the copy for this has an additional change that was made in the design doc reviews --
> 
> Instead of "Los gastos para el espacio de trabajo {workspace_name} están actualmente restringidos", we want to change the copy to "Las acciones en el espacio de trabajo {workspace_name} están actualmente restringidas"
> 
> @Gonals - are you able to confirm if that is the correct formatting to say, "Actions on the {workspace_name} workspace are currently restricted" ? I think we'd also need to change the English file for this as well? As initially the design doc had "Expenses to {workspace_name} are currently restricted."

Yep! All good!